events/static/events/create_event.css

```diff
@@ -0,0 +1,245 @@
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container-shadow {
+  position: absolute;
+  width: 600px;
+  height: 900px;
+  left: 48%;
+  margin-left: -262.5px;
+  margin-top: 10%;
+  box-shadow: 0px 30px 50px -20px #000;
+}
+
+.container {
+  position: absolute;
+  width: 600px;
+  height: 900px;
+  left: 48%;
+  margin-left: -262.5px;
+  margin-top: 10%;
+  background: linear-gradient(90deg, #29323c 0%, #3b4249 100%);
+  box-shadow: 0px 0px 50px -20px #000;
+}
+
+.wrap {
+  position: relative;
+  width: 400px;
+  margin: 0 auto;
+}
+
+.headings {
+  position: relative;
+  margin: 90px auto 70px;
+}
+
+.headings > a {
+  color: rgba(255, 255, 255, 0.7);
+  padding-bottom: 10px;
+  border-bottom: solid 2px rgba(17, 97, 237, 0);
+  display: inline-block;
+}
+
+.headings > a:first-child {
+  margin-left: 20px;
+  margin-right: 45px;
+}
+
+.headings > a:hover {
+  color: rgba(255, 255, 255, 1);
+}
+
+.headings > a.active {
+  color: rgba(255, 255, 255, 1);
+  border-bottom: solid 4px #aae0f0;
+  box-shadow: 0 5px 5px -5px #1d2124;
+}
+
+.headings > a span {
+  font-size: 1.4rem;
+  letter-spacing: 1px;
+}
+
+label {
+  color: rgba(255, 255, 255, 0.8);
+  text-transform: uppercase;
+  padding: 13px 20px;
+  position: relative;
+  display: block;
+  font-size: 14px;
+}
+
+input {
+  width: 100%;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  letter-spacing: 1px;
+  background: transparent;
+  padding: 10px 20px;
+  border: none;
+  border-bottom: 2px solid #1d2124;
+  outline: none;
+  box-sizing: border-box;
+
+  margin-bottom: 10px;
+}
+
+input:active,
+input:focus {
+  border-bottom: 0.12em solid #aae0f0;
+  border-color: #aae0f0;
+  box-shadow: 0 5px 5px -5px #aae0f0;
+}
+
+#title {
+  margin-top: -10px;
+}
+
+#input-title {
+  font-size: 1.3em;
+}
+
+#location {
+  font-size: 1.3em;
+}
+
+#id_color {
+  margin-left: 20px;
+  font-size: 1.3em;
+}
+
+#description {
+  width: 380px;
+  margin-left: 10px;
+  font-size: 1.3em;
+}
+
+#password + i {
+  color: rgba(255, 255, 255, 0.7);
+  font-size: 25px;
+  float: right;
+  position: relative;
+  top: -43px;
+  right: 15px;
+  cursor: pointer;
+}
+
+#password + i:hover {
+  color: rgba(255, 255, 255, 1);
+}
+
+input[type="checkbox"] {
+  width: inherit;
+  padding: 0;
+  margin: 20px 5px 30px 20px;
+}
+
+input[type="checkbox"] + span {
+  color: #fff;
+}
+
+.button {
+  width: 100%;
+  background: #aae0f0;
+  color: #2c353e;
+  text-transform: uppercase;
+  margin-top: 2em;
+  padding: 12px;
+  cursor: pointer;
+  border: 2px solid transparent;
+}
+
+.button:hover {
+  background: #aae0f0;
+  box-shadow: 0 0 10px #aae0f0, 0 0 30px #aae0f0, 0 0 50px #aae0f0;
+}
+
+footer .fp a {
+  color: rgba(255, 255, 255, 0.6);
+  cursor: pointer;
+}
+
+footer .hr {
+  border-bottom: 2px solid rgba(255, 255, 255, 0.1);
+  margin: 60px 0px 40px;
+}
+
+footer .fp {
+  text-align: center;
+}
+
+footer .fp a:hover {
+  color: rgba(255, 255, 255, 1);
+}
```

Comment:
Can you remove this CSS class if you don't use it in the rest of the HTML

Reply:
done